custom/succeed/models.py

```diff
@@ -107,4 +107,43 @@ class UCLAPatientFluff(fluff.IndicatorDocument):
 
     emitter = RandomizationDate()
 
+
+def get_full_name(case):
+    try:
+        return CommCareCase.get(case['indices'][0]['referenced_id'])['full_name']
+    except AttributeError:
+        return None
+
+
+class Emitter(fluff.Calculator):
+
+    @fluff.null_emitter
+    def date(self, case):
+        yield None
+
+
+class UCLATaskActivityFluff(fluff.IndicatorDocument):
+
+    document_class = CommCareCase
+    domains = 'succeed',
+    document_filter = CustomFilter(lambda c: c.type == 'task')
+    group_by = 'domain',
+    save_direct_to_sql = True
+
+    referenced_id = flat_field(lambda c: c['indices'][0]['referenced_id'])
+    full_name = flat_field(lambda c: get_full_name(c))
+    name = flat_field(lambda c: get_property(c, 'name').encode("iso-8859-15", "backslashreplace"))
```

Comment:
Curious why you are encoding this as `iso-8859-15`? This will make the `name` field a bytes object, which seems wrong, but I could be missing something. Also, the encoding we typically use is UTF-8, so I'm just pretty confused.